extensions/script-commands/src/Components/Common/FilterToast.tsx

```diff
@@ -0,0 +1,30 @@
+import { 
+  Toast, 
+  ToastStyle, 
+  showToast 
+} from "@raycast/api"
+
+import { 
+  Filter, 
+  State 
+} from "@types"
+
+export async function FilterToast(filter: Filter): Promise<Toast | null> {
+  if (filter == null)
+    return null
+
+  let title = "Filter activated: "
+  const message = `To clear filter, press Command + Shift + C`
+  const style = ToastStyle.Success
+
+  if (typeof filter == "string")
```

Comment:
You have a good point here. Even though I prefer to implement my conditionals omitting curly braces, here the code will be opened for the whole community and this may bring problems if someone decide to make changes not following my preferences. To prevent, it's better to surround the conditionals with curly braces! Thanks for the suggestion!